src/main/java/org/wildfly/security/credential/source/FactoryCredentialSource.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
```

Comment:
I think this probably should be 2017 as that is when this class was written.

Reply:
fixed (sorry, no idea why Idea shows me "18/01/18" in blame :D)